app/controllers/api/trainees/withdraws_controller.rb

```diff
@@ -5,12 +5,10 @@ module Trainees
     class WithdrawsController < Api::BaseController
       def create
         if withdraw_allowed?
-          if errored_params.any?
-            render_parameter_invalid(parameter_keys: errored_params)
-          else
-            withdraw_trainee
-
+          if withdraw_trainee
             render(json: { data: TraineeSerializer.new(trainee).as_json }, status: :accepted)
```

Comment:
Not part of this PR, but I'm wondering whether we should be using status 200 (OK) rather than 202 (accepted) here because accepted is only meant for async (I think).

Reply:
Yeah i think 200 should really be fine.